src/types.ts

```diff
@@ -208,10 +208,10 @@ interface PlayerItems {
 
 export interface FilterRule {
   character?: ("veteran" | "zealot" | "psyker" | "ogryn")[]
```

Comment:
Should this become `("veteran" | "zealot" | "psyker" | "ogryn")[] | "veteran" | "zealot" | "psyker" | "ogryn"`? If so, probably worth pulling classes out to a new type e.g.

```typescript

export const CLASS_TYPES = ["veteran", "zealot", "psyker", "ogryn"] as const
export type ClassType = typeof CLASS_TYPES[number]
```